db/warehouse/migrate/20230914004821_add_hmis_case_notes.rb

```diff
@@ -0,0 +1,16 @@
+class AddHmisCaseNotes < ActiveRecord::Migration[6.1]
+  def change
+    create_table(:CustomCaseNote) do |t|
+      t.string :CustomCaseNoteID, null: false
+      t.string :PersonalID, null: false, index: true
+      t.string :EnrollmentID, index: true
+      t.references :data_source, null: false, index: false
```

Comment:
why `index: false` on data source? not just I understand that one, just curious.

Reply:
Good question! On line 13 we create a compound index where data_source_id is the first key. Postgres will use that index just as it would a dedicated index on data_source_id, so the extra index would be redundant. Redundant indexes are less of a concern on small tables but can impact space/write-performance on larger tables so I try to keep an eye out for them

Reply:
That makes sense, thanks!